### ISSUE TITLE
Cannot remove /var/lib/docker (Read-only file system)

### ISSUE THREAD

**Issue by goofrider** [docker/docker#7773]
I'm trying to remove /var/lib/docker and start clean, but `rm -Rf /var/lib/docker/*` gave me a read-only error on almost EVERY subdirectory inside.

```
rm: cannot remove ‘rm: cannot remove ‘aufs/diff/798768cdbda5c22d7beecc438a5abd8ba02711721a08a334c700315c3d64552e/usr/lib/python2.7/mutex.pyc’: Read-only file system
rm: cannot remove ‘aufs/diff/798768cdbda5c22d7beecc438a5abd8ba02711721a08a334c700315c3d64552e/usr/lib/python2.7/netrc.py’: Read-only file system
rm: cannot remove ‘aufs/diff/798768cdbda5c22d7beecc438a5abd8ba02711721a08a334c700315c3d64552e/usr/lib/python2.7/netrc.pyc’: Read-only file system
rm: cannot remove ‘aufs/diff/798768cdbda5c22d7beecc438a5abd8ba02711721a08a334c700315c3d64552e/usr/lib/python2.7/new.py’: Read-only file system
...
```

My /var/lib/docker is actually mounted as a btrfs subvolume. But I could do `btrfs subvolume delete` either, it gives the same read-only file system error. Downgrading docker didn't help, and docker daemon wouldn't run at all, it just quits without much warning. Here's from the docker upstart log.

```
2014/08/28 14:09:24 docker daemon: 1.2.0 fa7b24f; execdriver: native; graphdriver: 
[91046fd4] +job serveapi(unix:///var/run/docker.sock)
[info] Listening for HTTP on unix (/var/run/docker.sock)
[91046fd4] +job init_networkdriver()
[91046fd4] -job init_networkdriver() = OK (0)
2014/08/28 14:09:24 remove /var/lib/docker/init/dockerinit-1.2.0: read-only file system
2014/08/28 14:09:24 docker daemon: 1.2.0 fa7b24f; execdriver: native; graphdriver: 
[f4a5d27e] +job serveapi(unix:///var/run/docker.sock)
[info] Listening for HTTP on unix (/var/run/docker.sock)
[f4a5d27e] +job init_networkdriver()
[f4a5d27e] -job init_networkdriver() = OK (0)
2014/08/28 14:09:24 remove /var/lib/docker/init/dockerinit-1.2.0: read-only file system
...
```

It jus pretty much rinse and repeat if I restart the daemon. Any pointers to what I can do next to troubleshoot this would be greatly appreciated.

I'm running Ubuntu 14.04, Kernel 3.13.0-34, Docker 1.2.0 and Btrfs v3.12.

**Comment by jfrazelle** [comment]
closing as duplicate of #9939 which is a better description of the issue

**Comment by mwaeckerlin** [comment]
Loop:
- #9939 closed as duplicate of #7773
- #7773 closed as duplicate of #9939

Please reopen and fix #9939!

I just had to cleanup due to the same reason!
